src/components/Atoms/AddTaskButton/index.jsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import plus from "../../../assets/svg/plus.svg";
+import styled from "styled-components";
+import FONT from "../../../variables/font_family";
+import COLOR from "../../../variables/color";
+import TEXT from "../../../variables/texts";
+
+export const TaskButton = (props) =>{
+    return (
+        <StyledTaskButton onClick={props.onClick}>
```

Comment:
```suggestion
export const TaskButton = ({onClick}) =>{
    return (
        <StyledTaskButton onClick={onClick}>
```
ワンポイントアドバイス的なやつです。
propsの渡し方として分割代入という方法があります。基本この形で渡されていることが多いので知っておいて損はないと思います。
参考文献
[MDN_分割代入](https://developer.mozilla.org/ja/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment)
[【React】関数コンポーネントの引数（props）は分割代入で渡すとわかりやすい](https://dezanari.com/react-component-props-object/)